docs/pages/cheatsheet.adoc

```diff
@@ -675,32 +763,21 @@ A group can contain only menu link or menu table elements.
 |hidden         | boolean expression    | If the menu group is shown to the user.
 |====================
 
-=== Actor menu link
+=== Actor menu
 
 *Syntax:*
 
 [subs="quotes"]
 ----
-[purple]#**link**# &lt;view> &lt;name> &#x5B;[purple]##**<=**## &lt;expression>]
+[purple]#**menu**# &lt;view> &lt;name> [purple]##**<=**## &lt;expression>
 &#x5B;&lt;parameter name> [purple]#**:**# &lt;parameter value>] ... [purple]#**;**#
 ----
 
-[options="header", cols="1,1,2"]
-|====================
-|Parameter name | Parameter type        | Meaning
-|label          | string literal        | The label to show for the menu.
-|icon           | string literal        | the name of the icon to show for the menu.
-|enabled        | boolean expression    | If the menu is enabled to the user.
-|hidden         | boolean expression    | If the menu is shown to the user.
-|====================
-
-=== Actor menu table
-
-*Syntax:*
+or
 
 [subs="quotes"]
 ----
-[purple]#**table**# &lt;row>[purple]##**[]**## &lt;name> [purple]##**<=**## &lt;expression>
+[purple]#**menu**# &lt;row>[purple]##**[]**## &lt;name> [purple]##**<=**## &lt;expression>
```

Comment:
Not sure if we talked about this or not, but right now I have no idea where would `menu <row>[] <name> <= <expression>` navigate to.